config/locales/server.en.yml

```
@@ -8,6 +8,8 @@ en:
     discourse_video_file_extensions: A list of file extensions which can be uploaded
     discourse_video_uploads_per_day_per_user: Maximum number of videos that a user can upload per day
     discourse_video_enable_mp4_download: "Allows users to download videos"
+    discourse_video_max_duration_minutes: Maximum number of minutes a trust level 1-3 user can upload
+    discourse_video_max_duration_minutes_leaders: Maximum number of minutes a trust level 4 user can upload
```

```suggestion
    discourse_video_max_duration_minutes: Maximum video duration (in minutes) a trust level 1-3 user can upload
    discourse_video_max_duration_minutes_leaders: Maximum video duration (in minutes) a trust level 4 user can upload
```

Just want to add the word "video" in there, to be super clear.